### ISSUE TITLE
TypeError when copying static files (TravisCI)

### ISSUE THREAD

**Issue by carlosdanielcsantos** [sphinx-doc/sphinx#4139]
Subject: TypeError with Sphinx + tox + tox-travis + TravisCI

### Problem
When running `sphinx-build -W -b html -d {envtmpdir}/doctrees .  {envtmpdir}/html` from a tox.ini that is called with TravisCI using tox-travis, a TypeError is raised when copying static files.

#### Procedure to reproduce the problem
.travis.yml

```
sudo: false
language: python
python:
  - "3.5"
install: pip install tox-travis
script:
  - tox -e docs
```

tox.ini

```
[testenv:docs]
basepython = python3.5
changedir = doc
deps = sphinx
commands =
    sphinx-build -W -b html -d {envtmpdir}/doctrees .  {envtmpdir}/html
    open {envtmpdir}/html/index.html
```

#### Error logs / results
```
writing output... [ 93%] user_guide
writing output... [100%] whats_new

generating indices...
writing additional pages... search
copying static files... 
Exception occurred:
  File "/home/travis/build/my_user/my_project/.tox/docs/lib/python3.5/site-packages/sphinx/util/logging.py", line 363, in filter
    raise SphinxWarning(message % record.args)
TypeError: not all arguments converted during string formatting
The full traceback has been saved in /tmp/sphinx-err-zpra8ul5.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!
[ERROR: InvocationError: '/home/travis/build/my_user/my_project/.tox/docs/bin/sphinx-build -W -b html -d /home/travis/build/my_user/my_project/.tox/docs/tmp/doctrees . /home/travis/build/my_user/my_project/.tox/docs/tmp/html'

```

#### Expected results
Just compiling cleanly, as when I call tox locally.

### Environment info
- OS: Ubuntu 14.04.5 LTS
- Python version: 3.5.3
- Sphinx version: 1.6.4

**Comment by carlosdanielcsantos** [comment]
Just tried with Sphinx 1.6.1 and this error does not occur!

**Comment by stephenfin** [comment]
This is a duplicate of #4070 and #4111. This was fixed in 50640b700b4cbc9b141a72765378d9a1e416bd22 but has not yet been included in a release. Guess you need to wait for Sphinx 1.6.5 to drop and exclude 1.6.4 (and maybe 1.6.3? idk) until that time:

    $ cat requirements.txt
    Sphinx>=1.6,<1.7,!=1.6.4
    ...

**Comment by tk0miya** [comment]
Yes, this is already fixed, but not released yet.
Please wait a moment. I will release 1.6.5 soon.
